README.md

```
@@ -86,6 +102,35 @@
 ### Continuous Integration
 The project employs GitHub Actions for continuous integration, ensuring that tests pass on every push.
 
+## 🖥️ TUI Keyboard Shortcuts
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)